cmd/kube_create.go

```diff
@@ -123,12 +123,12 @@ var kubeCreateCmd = &cobra.Command{
 		if err != nil {
 			// If the cluster instance doesn't exist (expected case) we receive a well deserved
 			// "431 not found" error, but check if we got any other error just in case...
-			if csError, ok := err.(*egoscale.ErrorResponse); ok && csError.ErrorCode != 431 {
+			if csError, ok := err.(*egoscale.ErrorResponse); ok && csError.ErrorCode != egoscale.ParamError {
 				return err
 			}
 		}
 		if vm != nil {
-			return fmt.Errorf("Cluster instance %q already exists", clusterName) // nolint: golint
+			return fmt.Errorf("cluster instance %q already exists", clusterName)
```

Comment:
Nitpicking, but I find error messages starting in lower case quite gross. Could we at least have a `Error:` prefix printed before the rest of the lowercase message?

Reply:
I favor consistency over personal taste including my own. The less `nolint` we have the better I feel.